MapboxNavigation/Resources/Base.lproj/Navigation.storyboard

```diff
@@ -247,12 +264,13 @@
                             <constraint firstItem="drb-do-FoT" firstAttribute="centerX" secondItem="bUe-nv-g0F" secondAttribute="centerX" id="O0m-0k-ilC"/>
                             <constraint firstItem="p9E-v0-Rt6" firstAttribute="bottom" secondItem="UnK-tl-lPe" secondAttribute="bottom" id="OJu-tK-ZSa"/>
                             <constraint firstItem="Gx9-Bj-wuQ" firstAttribute="leading" secondItem="Gvo-VD-DXF" secondAttribute="leading" id="QHw-st-QI3"/>
-                            <constraint firstItem="Gx9-Bj-wuQ" firstAttribute="top" secondItem="drb-do-FoT" secondAttribute="bottom" constant="10" id="SWf-LB-JLK"/>
```

Comment:
Looks like a merge conflict. This makes the Y position of the WayNameLabel ambiguous. Pin it 10pts from the bottom banner.

Reply:
- [ ] Fix Storyboard Merge Clobber